.github/workflows/ci.yml

```diff
@@ -11,17 +13,17 @@ jobs:
   build:
     runs-on: ${{ matrix.os }}
     strategy:
-      fail-fast: true
+      fail-fast: false
       matrix:
-        os: [ubuntu-latest, macOS-latest]
-        ghc: ['9.2', '9.0', '8.10', '8.8', '8.6', '8.4', '8.2']
+        os: [ubuntu-20.04, macOS-latest]
```

Comment:
Why not latest?

Reply:
GHC 8.2.2 has no install on `ubuntu-22.04`.  
See: https://github.com/andreasabel/unix/actions/runs/3624403077/jobs/6111349578
> Error: All install methods for ghc 8.2.2 failed

Reply:
It does, the haskell setup action just swallows the error: https://github.com/andreasabel/unix/actions/runs/3624403077/jobs/6111349578#step:3:50

The most common cause is different libncurses.

Try installing `libncurses5` and `libtinfo5`.

Reply:
> Try installing `libncurses5` and `libtinfo5`.

That helped, thanks!

Bumped matrix to `ubuntu-22.04` then.